cvmfs/cvmfs_config

```diff
@@ -245,6 +312,12 @@ cvmfs_chksetup() {
     fi
   done
 
+  # Check that CVMFS_USER is set
+  if [ x"$CVMFS_USER" = x"" ]; then
```

Comment:
In principle yes. However, I was getting unclear follow-up errors when `CVMFS_USER` was not set but used before the `required_list` was actually checked.